lib/ui/nav_pages/dm_page/dm_page.dart

```diff
@@ -1,13 +1,108 @@
 import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:hng/ui/shared/colors.dart';
+
+import 'package:stacked/stacked.dart';
+
+import 'dm_page_viewmodel.dart';
+import 'widgets/dmmessage_read.dart';
+import 'widgets/dmmessage_unread.dart';
+
 
 class DmPage extends StatelessWidget {
   const DmPage({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      alignment: Alignment.center,
-      child: const Text('DM Page'),
+    return ViewModelBuilder<DmPageViewModel>.reactive(
+      builder: (context, model, child) {
+        return Scaffold(
+          appBar: AppBar(
+            elevation: 0,
+            backgroundColor: Color(0xff00B87C),
+            title: Text(
+              "Direct Messages",
+              style: GoogleFonts.lato(
+                fontWeight: FontWeight.w700,
+              ),
+            ),
+            centerTitle: false,
+          ),
+          floatingActionButton: FloatingActionButton(
+            onPressed: () {},
+            child: const Icon(
+              Icons.add,
+              color: AppColors.whiteColor,
```

Comment:
The fab colors are being taken care of by ThemeService (theme_setup.dart)

Reply:
Ooh cool... I will take note. Thanks